tests/unit/controllers/users.js

```diff
@@ -143,18 +138,6 @@ exports['getType returns unknown when roles is empty'] = function(test) {
   test.done();
 };
 
-exports['getType returns admin when user is in admins list and roles is empty'] = function(test) {
-  var user = {
-    name: 'sam',
-    roles: []
-  };
-  var admins = {
-    'sam': 'x'
-  };
-  test.equal(controller._getType(user, admins), 'admin');
-  test.done();
-};
```

Comment:
This is the unit test we're gonna have to talk about re: the comment above. Since we don't want people using CouchDB admin users we don't really want to support this kind of thing. Question is: what is it enabling (ideally nothing, but you'll know more than me)